client/src/login/LoginContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { LoginForm } from './LoginForm';
+import { authenticateUser } from './LoginActions';
+
+function mapDispatchToProps(dispatch) {
+  return ({
+    onSubmit: (values) => {
+      dispatch(authenticateUser(values.email, values.password));
+    },
+  });
+}
+
+function mapStateToProps() {
+  return ({});
+}
+
+export const LoginContainer = connect(mapStateToProps, mapDispatchToProps)(LoginForm);
```

Comment:
I don't know how to create a class here and solve all my jslint errors. my on submit function doesn't need `this` so it shouldn't be a class method. Also, if I did do the submission as a class method, should it accept the dispatch function as a param and call the action itself? Or should it just return the action?